### PR TITLE
Changes for NetSuite tests

### PR DESCRIPTION
**Pull request by silvadf9**
Removed duplicated code on common.py (caused by merge conflict)
Updated local.yml to take into account the netsuite.crx file and commented the third_party_cookie_phaseout which was causing issues on SF adapter Updated qaXX.yml with the new needed fields (for Lab01 need to verify urls for adapters) Added new identifiers to sf_login_page.py for NetSuite test Added new identifiers to adapter_login_page.py for NetSuite test Added new steps to sf_login_steps.py for NetSuite test Added new step to adapter_login_steps.py to launch the adapter through a gherkin step instead of automatically on test initialization Created netsuite_scenarios.feature file
Created netsuite_scenarios.py file which is launching 1 browser window with adapter configured and 1 normal window Created test_check_netsuite_scenarios.py file

### CODE REVIEW

File: configuration/lab/qa01.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 configuration:
     login_url: https://qalogin01.five9lab.com/
+    #TODO: Check correct link for adapters on Lab01
     adapter_login_url: https://qaapp11d.five9lab.com/clients/integrations/adt.main.html
+    adapter_ns_login_url: qaapp11d.five9lab.com
```
Comment (iagolivino1) [inline configuration/lab/qa01.yml:5]: this url is for lab02

File: configuration/local.yml [hunk 2]
```diff
@@ -10,12 +10,13 @@ configuration:
     test_name: <any_name>
 
 chrome_flags:
-    third_party_cookie_phaseout: --test-third-party-cookie-phaseout
+    # third_party_cookie_phaseout: --test-third-party-cookie-phaseout
```
Comment (iagolivino1) [inline configuration/local.yml:13]: lets keep the "original" files and create new ones when we need to run with other configs

File: test/initialization/netsuite_scenarios.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+    # find the extension
+    for _driver in driver.DRIVERS:
+        driver_ = driver.DRIVERS.get(_driver).get('instance')
+        set_adapter_shortcut(driver_, "Five9Lab Plus Adapter for NetSuite")
+        set_adapter_url(driver_, set_adapter_shortcut(driver_, "Five9Lab Plus Adapter for NetSuite", True))
+        break
+
+# ----- SETUP ----- #
+def set_adapter_shortcut(driver_, extension_text, return_id=False):
```
Comment (iagolivino1) [inline test/initialization/netsuite_scenarios.py:40]: as you are using the same code, you can call this method direct from adt_plus_scenarios.py initialization module.
or we can create some common utils for adapter initialization.

File: test/initialization/netsuite_scenarios.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    edit_shortcut_button = edit_shortcut_shadow.shadow_root.find_element(By.CSS_SELECTOR, "cr-icon-button#edit")
+    edit_shortcut_button.click()
+    edit_shortcut_shadow.shadow_root.find_element(By.CSS_SELECTOR, "cr-input").shadow_root.find_element(By.CSS_SELECTOR, "#input").click()
+    action.key_down(adapter_login_steps.PLATFORM_HOTKEYS.get(driver_.caps.get('platformName'))[0])\
+        .send_keys('i')\
+        .key_up(adapter_login_steps.PLATFORM_HOTKEYS.get(driver_.caps.get('platformName'))[0])\
+        .perform()
+
+
+def set_adapter_url(driver_, extension_id):
```
Comment (iagolivino1) [inline test/initialization/netsuite_scenarios.py:87]: same here

File: configuration/local.yml (hunk 2 above)
Comment (silvadf9) [reply configuration/local.yml]: Changes reverted

File: test/initialization/netsuite_scenarios.py (hunk 3 above)
Comment (silvadf9) [reply test/initialization/netsuite_scenarios.py]: Please check new commit

File: test/initialization/netsuite_scenarios.py (hunk 4 above)
Comment (silvadf9) [reply test/initialization/netsuite_scenarios.py]: Please check new commit

**Review by iagolivino1 — APPROVED** [review]
very good!

File: page_objects/adapters/adapter_login_page.py [hunk 5]
```diff
@@ -4,14 +4,15 @@
...
+        self.adt_url = "https://qaapp01d.five9lab.com/clients/integrations/adt.main.html"
+        self.ns_url = "qaapp11d.five9lab.com"
         self.adapter_user_input = "//input[@id='username']"
         self.adapter_pass_input = "//input[@id='password']"
         self.adapter_login_button = "//button[contains(@id,'login_btn')]"
         self.force_login_button = "//button[@id='force-login']"
 
-    def open_page(self):
-        self.driver.get(self.url)
+    def open_page(self, url="https://qaapp01d.five9lab.com/clients/integrations/adt.main.html"):
```
[the remark this answers is not in the document]
Comment (silvadf9) [reply page_objects/adapters/adapter_login_page.py:13]: removed default

File: page_objects/sf_login_page.py [hunk 6]
```diff
@@ -3,17 +3,24 @@
...
+        self.logo_img = "//img[@id='phHeaderLogoImage'] | //div[@id='uif35']"
+        self.sf_url = "https://login.salesforce.com/"
+        self.ns_url = "https://system.netsuite.com/pages/customerlogin.jsp?country=US"
+        self.ns_additional_auth_title = "//div[@class='roleswitch-title']/h1"
+        self.ns_additional_pass = "//input[@name='answer']"
+        self.ns_additional_submit_btn = "//input[@name='submitter']"
 
-    def open_page(self):
-        self.driver.get(self.url)
+    def open_page(self, url="https://login.salesforce.com/"):
```
[the remark this answers is not in the document]
Comment (silvadf9) [reply page_objects/sf_login_page.py:16]: removed default

**Pull request merged**